modules/ROOT/pages/create-keystore.adoc

```diff
@@ -0,0 +1,45 @@
+= AS2 Host Keystore
+
+== Keystore Settings
+
+You can create a new keystore for the host when configuring AS2 receive and send endpoints. After creating the keystore, you can reuse it in new endpoint configurations.
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Required
+|Keystore name
+|The unique name for the keystore. The value in this field is automatically generated but can be modified.
+|Yes
+
+|<Host> public certificate
+|The digital certificate for your organization. This certificate is based on the RSA algorithm for the AS2 transport protocol.
+|Yes
+
+|<Host> private key file
+|The private key.
+|Yes
+
+|Key passphrase
+|The passphrase for your private key.
+|Yes
+|===
+
+== Create a Keystore
+. In the left navigation menu, select the host.
+. In the *Endpoints* section, click *New*. +
+The *New Endpoint* window appears.
```

Comment:
```suggestion
. In the *Endpoints* section, click *New*. +
+
The *New Endpoint* window appears.
```

Reply:
...but do we have to describe what the person performing the task can readily see?